client/src/components/Signup.js

```diff
@@ -0,0 +1,86 @@
+import React, { Component } from "react";
+import axios from "axios";
+import styled from "styled-components";
+
+const Container = styled.div`
+  display: flex;
+  width: 400px;
+  margin: 0 100px;
+  flex-direction: column;
+`;
+
+class Signup extends Component {
+  state = {
```

Comment:
This is awesome Rob! No constructor, you're using destructuring on line #37. This file shows you know your way around React, and es6 👍